Sources/StreamChatSwiftUI/ChatChannelList/MoreChannelActionsView.swift

```diff
@@ -91,6 +106,24 @@ public struct MoreChannelActionsView: View {
         .onTapGesture {
             onDismiss()
         }
+        .fullScreenCover(isPresented: $isPresented) {
+            if let fullScreenView = presentedView {
+                ZStack(alignment: .topTrailing) {
+                    fullScreenView
+                    
+                    Button {
+                        isPresented = false
+                        presentedView = nil
+                    } label: {
+                        Image(uiImage: images.closeFilled)
+                            .customizable()
+                            .frame(height: 30)
+                            .padding(15)
```

Comment:
We shouldn't use arbitrary numbers for paddings. If we have to, then the default ones should be applied (8,16, etc). In this case, only .padding() is enough, since it will apply the default 16 padding.